### PR TITLE
Moved point-to-integer to coding

### PR DESCRIPTION
**Pull request by rokuz**

### CODE REVIEW

File: coding/coding.pro [hunk 1]
```diff
@@ -68,6 +69,7 @@ HEADERS += \
     multilang_utf8_string.hpp \
     parse_xml.hpp \
     polymorph_reader.hpp \
+    point_to_integer.hpp \
```
Comment (mpimenov) [inline coding/coding.pro]: sort, please

**Comment by rokuz** [comment]
DUDE JTTP

File: indexer/indexer_tests/polyline_point_to_int64_test.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+namespace
+{
+  double const g_eps = MercatorBounds::GetCellID2PointAbsEpsilon();
+  uint32_t const g_coordBits = POINT_COORD_BITS;
+
+  void CheckEqualPoints(m2::PointD const & p1, m2::PointD const & p2)
+  {
+    TEST(fabs(p1.x - p2.x) < g_eps && fabs(p1.y - p2.y) < g_eps, (p1, p2));
```
Comment (mpimenov) [inline indexer/indexer_tests/polyline_point_to_int64_test.cpp]: my::AlmostEqualAbs

File: indexer/indexer_tests/polyline_point_to_int64_test.cpp [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+#include "base/logging.hpp"
+
+#include "std/cmath.hpp"
+#include "std/utility.hpp"
+
+
+namespace
+{
+  double const g_eps = MercatorBounds::GetCellID2PointAbsEpsilon();
```
Comment (ygorshenin) [inline indexer/indexer_tests/polyline_point_to_int64_test.cpp]: clang-format this file please.

**Comment by rokuz** [comment]
DUDE JTTP

**Pull request merged**